debugdialog.html

```diff
@@ -172,6 +172,11 @@
     return attributes
 }
 
+function getFeatureVariableType(featureKey, variableKey) {
+    var vmap = optimizelyClientInstance.getOptimizelyConfig().featuresMap[featureKey].variablesMap;
```

Comment:
I think you need to check if the feature key exists in the map first, otherwise you could run into a null reference exception when calling `.variablesMap`. If they key is not found, then just  return null or something